src/helpers/analytics.js

```diff
@@ -0,0 +1,59 @@
+// we could decide to have more specific helpers to make the code lighter for func templates
+// clickedObjectIDsAfterSearch(payload)
+// etc
+export function clickedObjectIDsAfterSearch(objectID, payload) {
```

Comment:
This is the helper layer, we only pass the data we want to override.
the exact payload is build by the connector layer so that it can benefit all the flavours.